Week3/DB_HW_Week3/index.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const mysql = require('mysql');
+
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'todo',
+});
+
+// Tables are created from MySQL command Line Client.
+
+// Error handler function.
+function errorHandler(err, res) {
+  if (err) {
+    console.log('Query did not execute:' + err);
+    res.sendStatus(500);
+  }
+}
+// 
+// 
+// Create New User
+app.post('/:userName/:userPassword', (req, res) => createUser(connection, errorHandler, req, res));
```

Comment:
Nice that you added support for multiple users 👍 
Although it is perfectly fine for this homework you should know that in real systems sensitive information is never sent as plain text in a URL. Also passwords are not stored as plain text in the DB :)